adapter/internal/oasparser/model/async_api.go

```diff
@@ -178,10 +178,9 @@ func populatePoliciesFromVendorExtensions(operation *Operation, vendorExtensions
 	policyParameters[constants.RewritePathResourcePath] = newResourcePath
 	policyParameters[constants.IncludeQueryParams] = true
 	policy := Policy{
-		PolicyName: constants.RewritePathResourcePath,
-		Action:     constants.RewritePathTemplate,
-		Order:      1,
-		Parameters: policyParameters,
+		Action:           constants.RewritePathAction,
+		Parameters:       policyParameters,
+		IsPassToEnforcer: true,
```

Comment:
Isn't this used for other API types? other than AsyncAPIs?

Reply:
`IsPassToEnforcer` is an internally used value, we are not getting this from API YAML or from outside, this is defined in the policy layout.
Fix hard coding true here by getting it from policy layouts with https://github.com/wso2/product-microgateway/pull/2805/commits/ff30a792424394e48e738a5927366d68925e70a8